docs/userGuide/plugins/texWithDollars.md

```diff
@@ -0,0 +1,40 @@
+### Plugin: TexWithDollars 
+
+This plugin allows you to use <tooltip content="$ ... $ or $$ ... $$">dollars</tooltip> as the delimiter for your math formulae.
+
+- Insert **inline** equations by enclosing them in single dollars `$...$`.
+
+- Insert **display** equations by enclosing them in double dollars `$$...$$`.
+
+- Insert numbered **display** equations by enclosing the equation in double dollars and the equation number in curly brackets `$$...$$ (1)`.
+
+<box type="info">
+
+Note that the **!!default delimiters still function as expected!!**, this plugin simply adds an additional delimiter. 
+
+</box>
+
+To enable this plugin, add `texWithDollars` to your site's plugins.  
+
+```js {heading="site.json"}
+{
+  ...
+  "plugins": [
+    "texWithDollars"
+  ]
+}
+```
+
+{{ icon_example }} Example:
+
+```markdown
+$2_3$
+
+$$
+\frac{1}{2}
+$$
+```
+The above will be rendered as:
+\(2_3\)
```

Comment:
Perhaps can add a line break here to bring the first example to a new line?

![image](https://user-images.githubusercontent.com/47470981/158056092-950384bf-ccd3-4f41-8c71-4dac0381a7c4.png)